tests/test_idutils.py

```diff
@@ -32,13 +32,16 @@
     ('http://www.example.org/ark:/13030/tqb3kh97gh8w', ['ark', 'url'], '', ''),
     ('10.1016/j.epsl.2011.11.037', ['doi', 'handle'],
         '10.1016/j.epsl.2011.11.037',
-        'http://dx.doi.org/10.1016/j.epsl.2011.11.037'),
```

Comment:
btw, the reason for the change is because it's the preferred way: https://www.doi.org/doi_handbook/3_Resolution.html#3.8